Model/Config.php

```diff
@@ -85,6 +87,16 @@ public function getPendingOrderTimeout()
         return (int) $this->getConfigData(self::PENDING_ORDER_TIMEOUT);
     }
 
+    public function isCancelResetCartOrderCronEnabled()
+    {
+        return (bool) (int) $this->getConfigData(self::ENABLE_RESET_CART_CRON, $this->storeId);
```

Comment:
why `bool` and `int` both?

Reply:
Data returned from table as `string` like '0' OR '1', need to return value as `bool`